src/test/java/org/folio/rest/api/LoanPoliciesApiTest.java

```diff
@@ -314,6 +314,93 @@ public void cannotCreateALoanPolicyWithAdditionalPropertiesInLoanPolicy()
       response.getStatusCode(), is(HttpStatus.HTTP_VALIDATION_ERROR.toInt()));
   }
 
+  @Test
+  public void canCreateLoanPolicyWithItemLimitWithinBounds()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException {
+
+    CompletableFuture<JsonResponse> createCompleted = new CompletableFuture<>();
+
+    UUID id = UUID.randomUUID();
+
+    JsonObject loanPolicyRequest = defaultRollingPolicy()
+      .withId(id)
+      .withName("Example Loan Policy")
+      .withDescription("An example loan policy")
+      .create();
+
+    loanPolicyRequest.getJsonObject("loansPolicy").put("itemLimit", 1000);
+
+    client.post(loanPolicyStorageUrl(),
+      loanPolicyRequest, StorageTestSuite.TENANT_ID,
+      ResponseHandler.json(createCompleted));
+
+    JsonResponse response = createCompleted.get(5, TimeUnit.SECONDS);
+
+    assertThat(String.format("Failed to create loan policy: %s", response.getBody()),
+      response.getStatusCode(), is(HttpURLConnection.HTTP_CREATED));
+  }
+
+  @Test
+  public void cannotCreateLoanPolicyWithItemLimitBelowMinimum()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException {
+
+    CompletableFuture<JsonResponse> createCompleted = new CompletableFuture<>();
+
+    UUID id = UUID.randomUUID();
+
+    JsonObject loanPolicyRequest = defaultRollingPolicy()
+      .withId(id)
+      .withName("Example Loan Policy")
+      .withDescription("An example loan policy")
+      .create();
+
+    loanPolicyRequest.getJsonObject("loansPolicy").put("itemLimit", 0);
+
+    client.post(loanPolicyStorageUrl(),
+      loanPolicyRequest, StorageTestSuite.TENANT_ID,
+      ResponseHandler.json(createCompleted));
+
+    JsonResponse response = createCompleted.get(5, TimeUnit.SECONDS);
+
+    assertThat(String.format("Should fail to create loan policy: %s", response.getBody()),
+      response.getStatusCode(), is(HttpStatus.HTTP_INTERNAL_SERVER_ERROR.toInt()));
```

Comment:
I think it make sense to assert error message as well, in order to make sure that the error is caused by validation rather then another error.

Hmm... another important note, why do we not expect validation error (422) in this case, but server error instead?

Could you please check if it is something related to https://issues.folio.org/browse/RMB-503 that @mreno-EBSCO observed? He has posted a temp fix for this issue.

Reply:
Adding this missing dependency did help, thanks!